jekyll-serve.sh

```diff
@@ -2,5 +2,4 @@
 
 set -e
 
-bundle exec jekyll build --incremental
```

Comment:
@josh-padnick added this recently as an optimization. Is this not necessary?

Reply:
AFAIK it works without it...